.gitignore

```diff
@@ -0,0 +1,5 @@
+src/main/java/com/google/codeu/servlets/AboutMeServlet.java
```

Comment:
These should be the files to exclude from tracking, so target/

Reply:
So instead of the files I currently have in .gitignore, it should just be "target/" to exclude all of the files from that folder?